### PR TITLE
Bandcamp parser

### PR DESCRIPTION
**Pull request by Greengrer**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -212,7 +212,13 @@
             <version>${json.simple.version}</version>
         </dependency>
 
-
+        <!-- AWAITILITY -->
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>4.1.0</version>
```
Comment (littlepony99) [inline pom.xml]: version to a constant

File: src/main/java/com/vinylteam/vinyl/util/impl/BandcampParser.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+import java.net.URLConnection;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+@Slf4j
+public class BandcampParser extends VinylParser {
+
+    private static final String BASE_LINK = "https://bandcamp.com/api/discover/3/get_web?g=all&s=top&p=&gn=0&f=vinyl&w=0&lo=true&lo_action_url=https%3A%2F%2Fbandcamp.com";
+    private static final String BASE_OFFER_LINK = "https://.bandcamp.com/album/";
```
Comment (littlepony99) [inline src/main/java/com/vinylteam/vinyl/util/impl/BandcampParser.java:29]: dot after https:// is ok?

File: src/main/java/com/vinylteam/vinyl/ScheduledTasksConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+@Configuration
+@EnableScheduling
+@Slf4j
+public class ScheduledTasksConfiguration {
+
+    @Autowired
+    private Updater updater;
+
+    //Equals to amount of milliseconds in one day
+    private static final long UPDATE_PERIOD = 518400000;
```
Comment (littlepony99) [inline src/main/java/com/vinylteam/vinyl/ScheduledTasksConfiguration.java]: 1000*60*60*24 will be more clear?

**Review by littlepony99 — CHANGES_REQUESTED** [review]

File: src/main/java/com/vinylteam/vinyl/util/impl/BandcampParser.java (hunk 2 above)
Comment (Greengrer) [reply src/main/java/com/vinylteam/vinyl/util/impl/BandcampParser.java:29]: Yes, I insert domain before that later!

File: pom.xml (hunk 1 above)
Comment (Greengrer) [reply pom.xml]: Thank you!

File: src/main/java/com/vinylteam/vinyl/ScheduledTasksConfiguration.java (hunk 3 above)
Comment (Greengrer) [reply src/main/java/com/vinylteam/vinyl/ScheduledTasksConfiguration.java]: Reasonable!

**Review by TEdwardJ — APPROVED** [review]

**Review by littlepony99 — APPROVED** [review]

**Pull request merged**